frontend/src/components/account/ContactDetails/ContactDetails.module.scss

```diff
@@ -1,27 +1,15 @@
 @import '../../../variables';
 
+.block {
+  padding-left: 40px;
+}
+
 .accountTitle {
   font-size: 20px;
   font-weight: 400;
```

Comment:
It is a default font-weight

Reply:
This style for h2, therefore I need font-weight